solr/heron/conf/solrconfig.xml

```diff
@@ -316,10 +316,26 @@
          uncommitted changes to the index, so use of a hard autoCommit
          is recommended (see below).
          "dir" - the target directory for transaction logs, defaults to the
-                solr data directory.  --> 
+                solr data directory.  -->
+    <!-- Not using transaction log due to issue upgrading from Solr v4_10_2 to v5_5_5:
```

Comment:
this is a heckuva find, and something to keep on a list of things to check for the next step.

Reply:
You are right! Some time this sprint I plan to add some sort of changelog to this repo, because I have a lot of notes on this PR's changes that I'd like to add in one place.